code/script.js

```diff
@@ -29,23 +31,154 @@ const showMessage = (message, sender) => {
+    showMessage(`Good choice! You selected ${selectedFood}.`, 'bot');
+    setTimeout(() => {
+      showTypeOfFood(selectedFood); 
+    }, 1000);
+  });
+});
+};
+
+//------------------------------ show type of food ------------------------------//
+
+const showTypeOfFood = (choice) => {
+  
+  if (choice === 'pizza') {
+    showMessage(`What type of pizza would you like?`, 'bot');
+    inputWrapper.innerHTML = `
+    <label for="food-select">Select your pizza:</label>
+    <select id="food-select" class="send-select">
+    <option disabled selected value> Select a pizza  🍕 🍝🥗 </option>
+      <option value="Margherita">Margherita</option>
+      <option value="Pepperoni">Pepperoni</option>
+      <option value="Hawaiian">Hawaiian</option>
+    </select>
+    `
+  } else if
+    (choice === 'pasta') {
+    showMessage(`What type of pasta would you like?`, 'bot');
+    inputWrapper.innerHTML = `
+    <label for="food-select">Select your pasta:</label>
+    <select id="food-select" class="send-select">
+    <option disabled selected value> Select a pasta 🍝</option>
+      <option value="Spaghetti">Spaghetti</option>
+      <option value="Alfredo">Alfredo</option>
+      <option value="Penne">Penne</option>
+    </select>
+    `
+  } else if
+    (choice === 'salad') {
+    showMessage(`What type of salad would you like?`, 'bot');
+    inputWrapper.innerHTML = `
+    <label for="food-select">Select your salad:</label>
+    <select id="food-select" class="send-select">
+    <option disabled selected value> Select a salad 🥗</option>
+      <option value="Caesar">Caesar</option>
+      <option value="Greek">Greek</option>
+      <option value="Garden">Garden</option>
+    </select>
+    `
+  } else {
+    showMessage(`You need chose again`, 'bot');
+  }
+  const foodSelect = inputWrapper.querySelector('#food-select');
+  foodSelect.addEventListener('change', (event) => {
+    const selectedTypeOfFood = event.target.value;
+    showMessage(` ${selectedTypeOfFood}.`, 'user');
+    showMessage(`You choose : ${selectedTypeOfFood}.`, 'bot');
+    setTimeout(() => {
+      selectAge(selectedTypeOfFood); 
+    }, 1000);
+  });
+}
+
+//------------------------------ Select age ------------------------------//
+
+const selectAge = () => {
+  showMessage(`Will that be for an adult or a child?`, 'bot');
+  inputWrapper.innerHTML = `
+  <button class="send-btn" id="Adult" type="submit">Adult</button>
+  <button class="send-btn" id="Child" type="submit">Child</button>
+`
+const ageButton = inputWrapper.querySelectorAll('.send-btn') // finds all elements with class "send-btn"
+ageButton.forEach (button => {
+  button.addEventListener('click', (event) => {
+    const selectedAge = event.target.id;
+    showMessage(` ${selectedAge}`, 'user');
+    if (selectedAge === "Adult"){
+      showMessage(`You selected ${selectedAge}.  It will be 130 SEK`, 'bot');
+    } else {
+      showMessage(`You selected ${selectedAge}.  It will be 100 SEK`, 'bot');
```

Comment:
Maybe change the phrasing a little to something like:
"You have selected that this order will be for a ${selectedAge}. That will be..."